util/search.js

```diff
@@ -133,6 +133,10 @@ export const handleFilterLibraries = ({
       return false;
     }
 
+    if (support.fireos && !library.fireos) {
+      return false;
+    }
+
```

Comment:
it'll be good to expand this in a follow up pr, as discussed previously, so that if a library is supported by android and doesn't require google play services then it will show up in this list too imo